es6-classes/circle.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(area, circumference, radius) {
```

Comment:
Per instruction:
Write its constructor to take the **circle's radius as a parameter** (and nothing else)

Reply:
![es6-classes2](https://user-images.githubusercontent.com/64106926/227423756-74314793-154b-450c-b04e-d4bc5f8f2034.png)